content/en/real_user_monitoring/rum_analytics.md

```diff
@@ -0,0 +1,134 @@
+---
+title: RUM Analytics
+kind: documentation
+description: ""
+further_reading:
+- link: "https://www.datadoghq.com/blog/dash-2019-new-feature-roundup/#real-user-monitoring"
+  tag: "Blog"
+  text: "Real User Monitoring"
+- link: "/real_user_monitoring/rum_explorer"
+  tag: "Documentation"
+  text: "Explore your views within Datadog"
+---
+
+## Overview
+
+Rum analytics extend the views explorer page with views data aggregation and split capabilities for troubleshooting and monitoring. You can control:
+
+* the query that filters the set of views to analyze
+* the dimensions over which to split data
+* the visualization method for aggregates and splits
+
+From an analytics visualization, you can, additionally:
+
+* create a widget in a dashboard out of that visualization
+* deep dive into subsets of the events list, depending on the interactions that the visualization enables
+
+## Build an analytics query
+
+Use the query to control what's displayed in your RUM Analytics:
+
+1. Choose a [Measure][1] or [Facet][2] to graph. [Measure][1] lets you choose the aggregation function whereas [Facet][2] displays the unique count.
```

Comment:
```suggestion
1. Choose a [measure][1] or [facet][2] to graph. [measure][1] lets you choose the aggregation function whereas [facet][2] displays the unique count.
```

Reply:
Thought: wouldn't the second measure remain capitalized as it is the first word in the sentence? I could be totally wrong on this depending on style.